pytests/func/test_breakpoints.py

```diff
@@ -57,3 +58,67 @@ def test_path_with_unicode(run_as, start_method):
 
         session.send_request('continue').wait_for_response(freeze=False)
         session.wait_for_exit()
+
+
+@pytest.mark.parametrize('condition_type, condition, value, hits', [
```

Comment:
Can you make this get dependent params from a dict? (so that they don't show up in the test name needlessly)